src/Rtp/SequenceTracker.h

```diff
@@ -0,0 +1,151 @@
+/**
+ * @file SequenceTracker.h
+ * @author Daniel Stiner (danstiner@gmail.com)
+ * @date 2021-12-08
+ * @copyright Copyright (c) 2021 Daniel Stiner
+ */
+
+#pragma once
+
+#include "ExtendedSequenceCounter.h"
+#include "Types.h"
+
+#include <chrono>
+#include <list>
+#include <map>
+#include <set>
+
+using namespace std::literals;
```

Comment:
I usually avoid `using namespace` in headers, since they'll be applied to any file that #includes this file.